src/ui/popup/manage-settings-page/sync-config.tsx

```diff
@@ -0,0 +1,20 @@
+import {m} from 'malevic';
+import {ViewProps} from '../types';
+import CheckButton from '../check-button';
+
+let synced = false;
```

Comment:
Oh no, don't store it globally, all the instances will share the same value. Instead add `syncSitesFixes` option.